mlflow/recipes/utils/tracking.py

```diff
@@ -30,6 +31,15 @@
 TrackingConfigType = TypeVar("TrackingConfig")
 
 
+def _get_run_name(run_name_prefix):
+    if run_name_prefix is None:
+        return None
+
+    sep = "-"
+    num = random.randint(0, 10**3)
```

Comment:
I think this is going to conflict. For example, you run your recipe with the same run name prefix twice and this line generates the same integer. Can we use a short hash like `fa18aj7a`?

Reply:
+1